cms/djangoapps/contentstore/views/xblock.py

```diff
@@ -21,7 +22,13 @@ def xblock_resource(request, block_type, uri):  # pylint: disable=unused-argumen
     """
     try:
         xblock_class = XBlock.load_class(block_type, select=settings.XBLOCK_SELECT_FUNCTION)
-        content = xblock_class.open_local_resource(uri)
+        # Note: in debug mode, return any file rather than going through the XBlock which
+        # will only return public files. This allows unbundled files to be served up
```

Comment:
That's a really good point. It turns out that these are views so they don't really fit in XBlock utils. Instead I've moved the CMS view to a new Django app under openedx.core.djangoapps.common_views, and I've changed both Studio and LMS to use it instead.